src/isl/mod.rs

```diff
@@ -30,66 +30,80 @@ pub mod isl_type_reference;
 #[cfg(test)]
 mod isl_tests {
     use crate::isl::isl_constraint::IslConstraint;
-    use crate::isl::isl_type::IslType;
+    use crate::isl::isl_type::{AnonymousIslType, IslType, NamedIslType};
     use crate::isl::isl_type_reference::IslTypeRef;
     use ion_rs::value::reader::element_reader;
     use ion_rs::value::reader::ElementReader;
     use ion_rs::IonType;
     use rstest::*;
-    use std::convert::TryInto;
 
-    // helper function to be used by isl tests
-    fn load(text: &str) -> IslType {
-        (&element_reader()
-            .read_one(text.as_bytes())
-            .expect("parsing failed unexpectedly"))
-            .try_into()
-            .unwrap()
+    // helper function to create NamedIslType for isl tests
+    fn load_named_type(text: &str) -> IslType {
+        IslType::NamedIslType(
+            NamedIslType::parse_from_owned_element(
+                &element_reader()
+                    .read_one(text.as_bytes())
+                    .expect("parsing failed unexpectedly"),
+            )
+            .unwrap(),
+        )
+    }
+
+    // helper function to create AnonymousIslType for isl tests
+    fn load_anonymous_type(text: &str) -> IslType {
+        IslType::AnonymousIslType(
+            AnonymousIslType::parse_from_owned_element(
+                &element_reader()
+                    .read_one(text.as_bytes())
+                    .expect("parsing failed unexpectedly"),
+            )
+            .unwrap(),
+        )
```

Comment:
question—the Kotlin implementation of `Schema` has a function [`newType(String)`](https://github.com/amzn/ion-schema-kotlin/blob/master/src/com/amazon/ionschema/Schema.kt#L78) that is similar to these ones. Would it be worth putting these functions in some other module where they could be reused by [`Schema`](https://github.com/amzn/ion-schema-rust/blob/main/src/schema.rs)?